<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSSReader</class>
 <widget class="QWidget" name="RSSReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="feedUrlLineEdit">
        <property name="placeholderText">
         <string>Input URL to add here</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QListWidget" name="feedListWidget"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTextEdit" name="feedContentTextEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addFeedButton">
        <property name="text">
         <string>Add RSS-feed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="removeFeedButton">
        <property name="text">
         <string>Remove feed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
